test/active_record_test.rb

```diff
@@ -172,4 +172,23 @@ def setup
       I18n.t('foo')
     end
   end
+
+  class WithCustomTranslationModel < I18nBackendActiveRecordTest
+    class CustomTranslation < I18n::Backend::ActiveRecord::Translation
+      def value=(v)
+        super("custom #{v}")
```

Comment:
Rubocop doesn't like short identifiers 🤷🏻 
```suggestion
      def value=(val)
        super("custom #{val}")
```